### PR TITLE
Fixed issue #17678: VV export didn't shown sub question

### PR DESCRIPTION
**Pull request by Shnoulle**
Dev: feature related to manage question text type and abbreviation

Fixed issue #<Mantis issue number>:
New feature #<Mantis issue number>:
Dev:

### CODE REVIEW

**Comment by Shnoulle** [comment]
Capture d’écran du 2022-03-03 19-41-56

File: application/views/admin/export/vv_view.php [hunk 1]
```diff
@@ -88,8 +78,30 @@
...
+                            <select class="form-control" name="qseparator" id="qseparator">
+                                <option value="newline"><?php eT("New line (use with care)"); ?></option>
+                                <option value="parenthesis" selected><?php eT("Sub question wrapped by parenthesis"); ?></option>
+                                <option value="dash"><?php printf(gT("Single dash (%s)"), ' - '); ?></option>
+                            </select>
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label for="abbreviatedtextto" class="col-sm-4 control-label">
+                            <?php eT("Separator for question and subquestion"); ?>
```
Comment (Shnoulle) [inline application/views/admin/export/vv_view.php:99]: Arg …

File: application/views/admin/export/vv_view.php [hunk 2]
```diff
@@ -88,8 +78,30 @@
                                     <?php eT("Old VV version"); ?>
                                 </label>
                             </div>
+                            <p class="help-block"><?php eT("If you want to import survey on old installation or if your survey have problem: use old version (automatically selected if some code are duplicated)."); ?></p>
```
Comment (c-schmitz) [inline application/views/admin/export/vv_view.php:81]: Please correct to:
"If you want to import the response data in older version or if your survey has integrity problem, please use the old export version (automatically selected if there are duplicate codes)."

File: application/views/admin/export/vv_view.php [hunk 3]
```diff
@@ -88,8 +78,30 @@
...
+                    </div>
 
+                    <div class="form-group">
+                        <label for="qseparator" class="col-sm-4 control-label">
+                            <?php eT("Separator for question and subquestion:"); ?>
+                        </label>
+                        <div class="col-sm-8">
+                            <select class="form-control" name="qseparator" id="qseparator">
+                                <option value="newline"><?php eT("New line (use with care)"); ?></option>
+                                <option value="parenthesis" selected><?php eT("Sub question wrapped by parenthesis"); ?></option>
```
Comment (c-schmitz) [inline application/views/admin/export/vv_view.php:92]: "Subquestion wrapped by parentheses"

File: application/views/admin/export/vv_view.php [hunk 4]
```diff
@@ -88,8 +78,30 @@
                                     <?php eT("Old VV version"); ?>
                                 </label>
                             </div>
+                            <p class="help-block"><?php eT("If you want to import survey on old installation or if your survey have problem: use old version (automatically selected if some code are duplicated)."); ?></p>
                         </div>
+                    </div>
 
+                    <div class="form-group">
+                        <label for="qseparator" class="col-sm-4 control-label">
+                            <?php eT("Separator for question and subquestion:"); ?>
```
Comment (c-schmitz) [inline application/views/admin/export/vv_view.php:87]: "Separator between question and subquestion:"

File: application/controllers/admin/Export.php [hunk 5]
```diff
@@ -686,43 +686,60 @@ public function vvexport()
...
                 } else {
                     $fieldcode = $field;
                 }
-                $secondline .= $fieldcode . $s;
+                $secondline[] = $fieldcode;
             }
-
-            $vvoutput = $firstline . "\n";
-            $vvoutput .= $secondline . "\n";
+            fputcsv($vvOutput, $firstline, "\t");
```
Comment (c-schmitz) [inline application/controllers/admin/Export.php:741]: Nice!

**Review by c-schmitz — CHANGES_REQUESTED** [review]

**Pull request merged**